### PR TITLE
docs(www): make .babelrc example valid JSON

### PR DESCRIPTION
**Pull request by ilyalesik**
## Description

JSON at [https://www.gatsbyjs.org/docs/babel/](https://www.gatsbyjs.org/docs/babel/) is invalid. PR fixes it.

### CODE REVIEW

**Comment by ilyalesik** [comment]
I wrap this code in prettier ignore section.

File: docs/docs/babel.md [hunk 1]
```diff
@@ -28,19 +28,21 @@ to add custom Babel presets or plugins, you can create your own `.babelrc` at th
...
       "babel-preset-gatsby",
       {
-        targets: {
-          browsers: [">0.25%", "not dead"],
-        },
-      },
-    ],
-  ],
+        "targets": {
+          "browsers": [">0.25%", "not dead"],
```
Comment (ZYSzys) [inline docs/docs/babel.md]: One more nit:

```suggestion
          "browsers": [">0.25%", "not dead"]
```

Comment (ilyalesik) [reply docs/docs/babel.md]: Fix this

**Comment by wardpeet** [comment]
In JSON5 this format is supported. Do you get any errors when processing this json?

Babel is using json5 to read the .babelrc file so this shouldn't be an issue https://babeljs.io/docs/en/config-files#json5

**Comment by ilyalesik** [comment]
WebStorm (and maybe other IDE) highlights filed names without quotes (fields, targets) and commas at .babelrc as syntax error. But maybe it's problem of this IDEs.

**Comment by wardpeet** [comment]
True, you need some extra config to let webstorm, vscode & other IDEs to actually understand it correctly. I'm in favour of moving to your solution.

I'm going to cc our @gatsbyjs/ecosystem here, just to make sure they are ok with it.

File: docs/docs/babel.md [hunk 2]
```diff
@@ -28,19 +28,21 @@ to add custom Babel presets or plugins, you can create your own `.babelrc` at th
 npm install --save-dev babel-preset-gatsby
 ```
 
+<!-- prettier-ignore-start -->
 ```json5:title=.babelrc
```
Comment (wardpeet) [inline docs/docs/babel.md]:
```suggestion
```json:title=.babelrc
```
Move to regular json syntax here. Please wait for this change until @gatsbyjs/ecosystem has answered.

**Pull request merged**